entity-framework/core/what-is-new/ef-core-8.0/whatsnew.md

```diff
@@ -493,6 +493,181 @@ CREATE TABLE [Documents] (
 > [!TIP]
 > Fibonacci numbers are used to limit the number of times a migration is generated to change the column length as new types are added to the hierarchy.
 
+### DateOnly/TimeOnly supported on SQL Server
+
+The <xref:System.DateOnly> and <xref:System.TimeOnly> types were introduced in .NET 6 and have been supported for several database providers (e.g. SQLite, MySQL, and PostgreSQL) since their introduction. For SQL Server, the recent release of a [Microsoft.Data.SqlClient](https://www.nuget.org/packages/Microsoft.Data.SqlClient/) package targeting .NET 6 has allowed [ErikEJ to add support for these types at the ADO.NET level](https://github.com/dotnet/SqlClient/pull/1813). This in turn paved the way for support in EF8 for `DateOnly` and `TimeOnly` as properties in entity types.
+
```

Comment:
Mention support is also available for EF Core 6/7 ? :smile:

Reply:
Yeah, a note seems to be a good idea.

Reply:
Done